keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -757,11 +760,12 @@ spec:
   queueLengthLimit: 100
 ```
 
-For requests from controllers processing workload.
+For requests from controllers processing workload, and anything else
+that does not go in a logically higher priority.
 ```yaml
 kind: RequestPriority
```

Comment:
These will need some updating after our API change goes in--doesn't have to happen in this PR.

Reply:
Note that these were designed before even the first draft of the API object types was written, and that took a direction not quite consistent with the ideas here.  Some of the predicates here can not be expressed with our current API object types.  So we have bigger fish to fry than just reformatting.

Reply:
https://github.com/kubernetes/kubernetes/pull/81637 is one of them.